src/styles/page-template/index.md.njk

```diff
@@ -16,11 +16,11 @@ This page template combines the boilerplate markup and [components](../../compon
 - the [skip link](../../components/skip-link), [header](../../components/header) and [footer](../../components/footer) components
 - the favicon, and other related theme icons
 
-In the examples provided, we show both HTML and Nunjucks.
+In the examples provided, we show both HTML and [Nunjucks](https://frontend.design-system.service.gov.uk/use-nunjucks-macros/).
 
 You can use the HTML examples if you are not able to use Nunjucks. If you use HTML you’ll need to update it manually when new versions are released.
```

Comment:
Again this seems like a place where we could talk about having that abstraction layer, so you only have to update that and not every page in your service.

Reply:
Sounds good but see above - might be something for next sprint?